caffe2/contrib/aten/CMakeLists.txt

```diff
@@ -3,17 +3,18 @@ if(BUILD_ATEN)
   # Unfortunately the custom command below requires this to be here
   set(__caffe2_CMAKE_POSITION_INDEPENDENT_CODE ${CMAKE_POSITION_INDEPENDENT_CODE})
   set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+  set(ATEN_NO_TEST ON)
   set(AT_LINK_STYLE STATIC)
-  add_subdirectory(${PROJECT_SOURCE_DIR}/aten aten)
+  add_subdirectory(../../../aten aten)
```

Comment:
Could this also be prefixed by ${CMAKE_CURRENT_SOURCE_DIR} ?

Reply:
I can add it to be similar to the previous version.